nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessPropagation.java

```diff
@@ -1011,6 +1012,25 @@ Nullness returnValueNullness(
         || !Nullness.hasNullableAnnotation((Symbol) node.getTarget().getMethod(), config)) {
       // definite non-null return
       nullness = NONNULL;
+      // check only when in JSpecify mode and the method is an interface method
+      if (node != null && config.isJSpecifyMode()) {
+        if (node.getTarget() != null && node.getTarget().getMethod() != null) {
+          if (node.getTarget().getMethod() instanceof Symbol.MethodSymbol) {
+            if (((Symbol.MethodSymbol) node.getTarget().getMethod()).owner.isInterface()) {
```

Comment:
We should not need to check for an interface method.  The method could be in a class as well.